staging/vhost-device-console/src/vhu_console.rs

```diff
@@ -105,10 +109,15 @@ pub(crate) struct VhostUserConsoleBackend {
     acked_features: u64,
     event_idx: bool,
     rx_ctrl_fifo: Queue<VirtioConsoleControl>,
-    rx_data_fifo: Queue<String>,
+    rx_data_fifo: Queue<u8>,
```

Comment:
Just for curiosity, why we changed from `String` to `u8`?

Reply:
The reason is that in the current implementation, from both stdin and tcpstream we retrieving the data already in bytes, and also the virtqueue_tx expects the data in bytes too. So I we don't need anymore to do any conversion between bytes and strings.